test/apis/v1beta1/api_validation_integration_test.go

```diff
@@ -179,6 +179,105 @@ var _ = Describe("Test networking v1alpha1 API validation", func() {
 			Expect(hubClient.Delete(ctx, profile)).Should(Succeed(), "failed to delete trafficManagerProfile")
 		})
 
+		It("should deny creating API with timeoutInSeconds < 5 when intervalInSeconds is 30", func() {
+			profile := &fleetnetv1beta1.TrafficManagerProfile{
+				ObjectMeta: objectMetaWithNameValid,
+				Spec: fleetnetv1beta1.TrafficManagerProfileSpec{
+					MonitorConfig: &fleetnetv1beta1.MonitorConfig{
+						IntervalInSeconds: ptr.To(int64(30)),
+						TimeoutInSeconds:  ptr.To(int64(4)),
+					},
+					ResourceGroup: "test-resource-group",
+				},
+			}
+			By("expecting denial of CREATE API with invalid timeoutInSeconds")
+			var err = hubClient.Create(ctx, profile)
+			Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create API call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8serrors.StatusError{})))
+			Expect(statusErr.Status().Message).Should(ContainSubstring("timeoutInSeconds must be between 5 and 10 when intervalInSeconds is 30"))
```

Comment:
could you please validate that, error should also contain "spec.monitorConfig.timeoutInSeconds in body should be greater than or equal to 5"